src/operations/edit/editorToCommand.ts

```diff
@@ -37,10 +38,10 @@ function defaultDetails(name: string): EditorCommandDetails {
  * @param {string} details object allowing customization beyond reasonable defaults
  * @return {HandleCommand}
  */
-export function editorHandler<PARAMS extends AllReposByDefaultParameters>(pe: (params: PARAMS) => AnyProjectEditor,
-                                                                          factory: ParametersConstructor<PARAMS>,
-                                                                          name: string,
-                                                                          details: Partial<EditorCommandDetails> = {}): HandleCommand {
+export function editorHandler<PARAMS extends MappedRepoParameters>(pe: (params: PARAMS) => AnyProjectEditor,
```

Comment:
shouldn't I be able to choose here? why must it used MappedRepoParameters? expected the superclass, GitHubParameters, here

Reply:
Yes you're correct